loadable_apps/loadable_sample/wifiapp/binary_update.c

```diff
@@ -644,26 +640,16 @@ void binary_update_test(void)
  ****************************************************************************/
 void binary_update_aging_test(void)
 {
-	int ret;
-	binary_update_info_t bin_info;
+	int randval;
 
 	while (1) {
-		ret = binary_update_getinfo(APP_NAME, &bin_info);
-		if (ret != OK) {
-			return;
-		}
-
-		/* Copy current binary with a higher version and valid crc. */
-		ret = binary_update_download_binary(&bin_info, DOWNLOAD_VALID_BIN);
-		if (ret != OK) {
-			return;
-		}
-
-		sleep(5);
-
-		ret = binary_update_reload(APP_NAME);
-		if (ret != OK) {
-			return;
+		randval = rand();
+		if (randval % 2 == 0) {
+			/* Iterate 'Micom' binary update */
```

Comment:
This sample iterates the binary update by randomly, but in the comments it looks that it iterates micom binary update only.